### DISCUSSION TITLE
Spices do not have licenses

### DISCUSSION THREAD

**Discussion by carlosmintfan** [linuxmint/discussions#641] [Questions] [answered]
Some or many spices in https://github.com/linuxmint/cinnamon-spices-actions (and possibly other spice types too, like applets) have no license, which is [too bad](https://choosealicense.com/no-permission).

**Comment by carlosmintfan** [comment, marked as answer]
[Discussed at the matrix](https://matrix.to/#/!sxwkYMHZAFBBLgzlFX:matrix.org/$DjNqatlMjYQu4RwBLfNivNyJrsBe8LRbpJOgDTtcpMI?via=matrix.org&via=envs.net&via=tchncs.de) and solved.

**Comment by rcalixte** [comment]
The Cinnamon license file (GPLv2) has been added to the root of the Spices repositories:
* [Actions](https://github.com/linuxmint/cinnamon-spices-actions/blob/9aed201/COPYING)
* [Applets](https://github.com/linuxmint/cinnamon-spices-applets/blob/fc92b61/COPYING)
* [Desklets](https://github.com/linuxmint/cinnamon-spices-desklets/blob/e86a104/COPYING)
* [Extensions](https://github.com/linuxmint/cinnamon-spices-extensions/blob/6b88066/COPYING)
* [Themes](https://github.com/linuxmint/cinnamon-spices-themes/blob/504a7ee/COPYING)


For authors who wish to use another license, they may include it with their contribution. All rights are included in the READMEs as well.
